propagator/google_cloud_platform/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History: opentelemetry-propagator-google_cloud_platform
+
+### v0.0.1 / 2025-02-11
+
+* Initial release.
```

Comment:
Our release tooling will generate the first changelog entry. No need to have anything besides the title.
```suggestion
```